### PR TITLE
Sandworm & Wizard Spider Emulation Plans

### PR DESCRIPTION
**Pull request by cat-alyst**
Co-authored-by: [Michael Long](https://github.com/bluesentinelsec)
Co-authored-by: [Daniel Matthews](https://github.com/uruwhy)
Co-authored-by: [Emmanuelle Vargas-Gonzalez](https://github.com/emmanvg)
Co-authored-by: [Brent Harrell](https://github.com/BCHarrell)
Co-authored-by: [Zoe Cheuvront](https://github.com/zcheuvront)
Co-authored-by: [Jamie Williams](https://github.com/jcwilliamsATmitre)
Co-authored-by: [Jesse Burgoon](https://github.com/jcburgoon)
Co-authored-by: [Jake Spinney](https://github.com/jspinney-MITRE)
Co-authored-by: [Sean Pagano](https://github.com/seanjpagano)
Co-authored-by: [Ian M. Davila](https://github.com/IanDavila)

Thank you all!

Order does not reflect contribution significance.

### CODE REVIEW

**Review by mehaase — CHANGES_REQUESTED** [review]
Due to the size I didn't look at all the code in great depth, but I did skim pieces of each component. A couple issues worth fixing before merge:

1. Many of the links and image URLs are absolute (leading `/`) instead of relative (leading `./`) and don't work when viewing on GitHub. E.g. look at [the sandword infrastructure](https://github.com/center-for-threat-informed-defense/adversary_emulation_library/blob/evalsR4/sandworm/Emulation_Plan/Scenario_1/Infrastructure.md): the two links in the resources section don't work, and the network diagram at the very end doesn't work either.
2. Don't check in build artifacts. These are usually created at checkout or build time. By removing these, the repository will be smaller (and PRs will be shorter, too).
    - packages/ directories for c# projects
    - __pycache__ for python projects
3. A few areas where there's a "todo" in the comments. We delete the comments and move them to GitHub issues (if we still want to do those things).

I made a few other remarks about things that we might want to restructure in the future, but those are just discussion points and shouldn't block this PR.

File: sandworm/Emulation_Plan/yaml/sandworm.yaml
Comment (cat-alyst) [reply sandworm/Emulation_Plan/yaml/sandworm.yaml:1]: This is a great idea for future editions, for this release let's leave it as is.

File: sandworm/Resources/Exaramel-Windows-Dropper/make.bat
Comment (cat-alyst) [reply sandworm/Resources/Exaramel-Windows-Dropper/make.bat:1]: I would love to build out a pipeline that aggregates all of the binaries into a password protected zip folder. Great idea for Adversary Emulation Library 2.0.

File: sandworm/Resources/NotPetya/SharpNP/packages/DllExport.1.7.4/.gitattributes
Comment (cat-alyst) [reply sandworm/Resources/NotPetya/SharpNP/packages/DllExport.1.7.4/.gitattributes:1]: Great point. I wasn't there for the code creation on this one. Let's leave this since it's small (don't want to break anything) and keep this use case in mind for the next evaluation merged. I am a fan of smaller PRs and clear instructions on builds.

File: sandworm/Resources/images/Sandworm Protections.drawio
Comment (cat-alyst) [reply sandworm/Resources/images/Sandworm Protections.drawio:1]: We had some great developers on this project. Glad you called this out.

File: wizard_spider/Emulation_Plan/Scenario_2/Infrastructure.md
Comment (cat-alyst) [reply wizard_spider/Emulation_Plan/Scenario_2/Infrastructure.md:4]: Good catch, this is wizard_spider that git is translating to this character. I'll update it with links to the group.

File: wizard_spider/Resources/control_server/evalsC2client.py
Comment (cat-alyst) [reply wizard_spider/Resources/control_server/evalsC2client.py:7]: I'll delete the TODOs from our tests/code. There are some TODOs that are pulled in from open source projects, I'll leave those.

File: sandworm/Resources/Exaramel-Windows-Dropper/README.md
Comment (cat-alyst) [reply sandworm/Resources/Exaramel-Windows-Dropper/README.md]: Resolved in [e0755a0](https://github.com/center-for-threat-informed-defense/adversary_emulation_library/pull/99/commits/e0755a080aae68058f7e00e80204c02483d1cbc1)

**Review by mehaase — APPROVED** [review]

**Pull request merged**
